tests/xharness/AppRunner.cs

```diff
@@ -487,6 +496,7 @@ public async Task<int> RunAsync ()
 			if (useXmlOutput) {
 				args.Append (" -setenv=NUNIT_ENABLE_XML_OUTPUT=true");
 				args.Append (" -setenv=NUNIT_ENABLE_XML_MODE=wrapped");
+				args.Append ("-setenv=NUNIT_XML_VERSION=nunitv3");
```

Comment:
missing space ? is that not concatenated with the previous one ?

Reply:
heh good catch!

Reply:
@spouliot nice! yeah, xcode11.4 does not have the new args add code, I missed that

Reply:
@spouliot missing space comes from a manual backport, that was the line that was giving the conflict. Original PR is ok, does not use a string builder.